.github/actions/install_requirements/action.yml

```diff
@@ -32,7 +32,7 @@ runs:
         mkdir -p lockfiles
         pip freeze --exclude-editable > lockfiles/${{ inputs.requirements_file }}
         # delete the self referencing line and make sure it isn't blank
-        sed -i '/file:/d' lockfiles/${{ inputs.requirements_file }}
+        sed -i.bak '/file:/d' lockfiles/${{ inputs.requirements_file }}
```

Comment:
I have no idea what this actually does?

Reply:
Fun little quirk: `sed` works slightly differently on different OS... Some will require a backup file. I had to test for a while but this is a version which will work on all of them.

Reply:
Change is in https://github.com/DiamondLightSource/python3-pip-skeleton/pull/143, I cherry-picked it here.